build.gradle.kts

```diff
@@ -32,11 +32,11 @@ config {
     release = false
 
     info {
-        group = "io.saagie"
         name = "Updatarium"
         description = "Automated update script management"
         inceptionYear = "2019"
         vendor = "Saagie"
+        group = "io.saagie.updatarium"
```

Comment:
I don't understand this modification because it's already in master branch : 
https://github.com/PierreLeresteux/updatarium/blob/master/build.gradle.kts#L40